Assets/HoloToolkit/Utilities/Shaders/FastConfigurable/Shaders/HoloToolkitCommon.cginc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#ifndef HOLOTOOLKIT_COMMON
+#define HOLOTOOLKIT_COMMON
+
+float4 _NearPlaneFadeDistance;
+
+// Helper function for focal plane fading
+// Could instead be done non-linear in projected space for speed
+inline float ComputeNearPlaneFadeLinear(float4 vertex)
```

Comment:
How come we are duplicating this code? HoloToolkitCommon.cginc still exists in Assets/HoloToolkit/Utilities/Shaders